spec/glueby/contract/payment_spec.rb

```diff
@@ -57,7 +57,7 @@
   end
 
   describe '#transfer' do
-    subject { Glueby::Contract::Payment.transfer(sender: sender, receiver: receiver, amount: amount) }
+    subject { Glueby::Contract::Payment.transfer(sender: sender, receiver_address: receiver.internal_wallet.receive_address, amount: amount) }
```

Comment:
Let's define the address directly in `let(:receiver)` rather than call `receiver.internal_wallet.receive_address
`.